benchmark/main.cpp

```diff
@@ -795,12 +795,7 @@ MBED_NOINLINE static int benchmark_ctr_drbg()
     mbedtls_ctr_drbg_init(&ctr_drbg);
 
     ret = mbedtls_ctr_drbg_seed(&ctr_drbg, myrand, NULL, NULL, 0);
-    if (ret == MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE) {
-        /* Do not consider this as a failure */
-        mbedtls_printf(HEADER_FORMAT "Feature unavailable\n", nopr_title);
-        mbedtls_printf(HEADER_FORMAT "Feature unavailable\n", pr_title);
-        goto exit;
-    } else if (ret != 0) {
+    if (ret != 0) {
```

Comment:
I don't mind it handled this way, but for the consistency's sake I'd consider leaving the alternative, having both branches fail, but print different error messages (if that makes sense from the functional standpoint).

Reply:
I see what you mean, but IMHO this is already implemented. Note that PRINT_ERROR actually prints the output of mbedtls_strerror() so there will be a different error message that clearly states whether FEATURE_UNAVAILABLE is the cause of the problem. I think it is redundant to further split this.